src/fetchtastic/repo_downloader.py

```
@@ -255,7 +255,9 @@ def main(config):  # log_message_func removed
                         or "y"
                     )
                     if open_folder == "y":
-                        os.startfile(download_folder)  # nosec B606
+                        os.startfile(
```

Ruff (0.14.8)

Starting a process without a shell

(S606)